text/0000-long-error-codes-explanation-normalization.md

```diff
@@ -0,0 +1,143 @@
+
+Start Date: 2016-01-04
+
+RFC PR:
+
+Rust Issue: N/A
+
+# Summary
+
+Long error codes explanations haven't been normalized yet. For example, not all error code's explanations provide an erroneous code example or a way to fix the error. This RFC intends to clarify long error codes by providing a template to follow.
```

Comment:
I would change this first bit to be a bit more clear about what "normalization" means. Something like this?

> Rust has extended error messages that explain each error in more detail. We've been writing lots of them, which is good, but they're written in different styles, which is bad. This RFC intends to fix this inconsistency by providing a template for these long-form explanations to follow.

WDYT?